tailwind.config.ts

```diff
@@ -2,7 +2,15 @@
 module.exports = {
   content: ["./src/**/*.{ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        "light-yellow": "#FCE490",
+        "dark-cyan": "#023851",
```

Comment:
you have a merge conflict, feel free to solve it. we'll go over the naming style we want at tomorrow's meeting but generally we go from dev-station-yellow-100, and the numbers go up as the colors get darker